modules/rules_pkl/0.6.0/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/apple/rules_pkl/releases/download/v0.6.0/rules_pkl-0.6.0.tar.gz",
+    "integrity": "sha256-wElnzxa4s6IPLj2GNddT5l/HIkD83kTtcvvtzqioQmo=",
+    "strip_prefix": "rules_pkl-0.6.0",
+    "patches": {
+        "update-version.patch": "sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU="
```

Comment:
Do we need an empty `.patch` file?

Reply:
This was automatically added by the `add_module` script. I can remove it.